src/cmd/flux-job.c

```diff
@@ -134,6 +158,9 @@ static struct optparse_option attach_opts[] =  {
     { .name = "quiet", .key = 'q', .has_arg = 0,
       .usage = "Suppress warnings written to stderr from flux-job",
     },
+    { .name = "debug-emulate", .key = 'd', .has_arg = 0,
+      .usage = "Set MPIR_being_debugged for testing",
```

Comment:
We probably don't want to take up a short option `-d` for a test-only option. 

Also, consider `.flags = OPTPARSE_OPT_HIDDEN` so the option doesn't appear in `flux job attach --help` otuput.

Reply:
Ah... I was wondering why some options have short and long options while others only long. This explains it. I will change it (along with `OPTPARSE_OPT_HIDDEN`).